### PR TITLE
Superglue audit

### PR DESCRIPTION
**Pull request by Holli-Git**
#### Summary
Content "Superglue spawns more"

#### Purpose of change

This is being closed, and replaced by [#75682](https://github.com/CleverRaven/Cataclysm-DDA/pull/75682) 
Superglue currently only spawns in single use tubes, while IRL superglue can last a lot longer than one use. 

#### Describe the solution

Superglue now spawns in tubes/bottles, which differing amounts. Also tweaked some recipes to make more sense, and added some requirements that should hopefully provide a basis for more glues, like basic white glue, wood glue, etc. 

#### Describe alternatives you've considered

Not doing this.

#### Testing

image
image
No errors on load 

#### Additional context

Superglue is in a bit of a weird spot, as since its kind of our only glue, a lot of recipes have it required in huge amounts, such as entry suits. I'm unaware of any good solutions to this tbh, though a lot of the problem recipes such as the aforementioned entry suit have issues with them. 

Current issues: Superglue can be transported out of their large containers. When updating, superglue will spill due to it being changed to a liquid. Innawoods does't like the migration

### CODE REVIEW

File: data/json/items/generic.json [hunk 1]
```diff
@@ -311,18 +311,22 @@
...
     "id": "superglue",
     "category": "spare_parts",
     "symbol": ",",
+    "phase": "liquid",
     "color": "white",
     "name": { "str_sp": "superglue" },
     "description": "A tube of strong glue.  Used in many crafting recipes.",
     "price": "18 USD",
     "price_postapoc": "1 USD",
     "material": [ "plastic" ],
```
Comment (Karol1223) [inline data/json/items/generic.json:324]: You probably want to make it no longer plastic if you're making it the liquid and not the entire bottle

File: data/json/items/generic.json [hunk 2]
```diff
@@ -311,18 +311,22 @@
...
-    "weight": "8 g",
-    "volume": "10 ml",
+    "weight": "1 g",
+    "volume": "1 ml",
+    "ammo_type": "superglue",
+    "//": "1 charge of superglue should cover an area of 0.5mm^2",
+    "count": 40,
```
Comment (Karol1223) [inline data/json/items/generic.json:329]: You do realize that with this, a single 'charge' of superglue is 0.025ml, right?

**Review by Karol1223 — CHANGES_REQUESTED** [review]

**Comment by Hydrogen-Deuteride** [comment]
> Superglue currently only spawns in single use tubes, while IRL superglue can last a lot longer than one use.

If by "super glue" means some sort of cyanoacrylate glue(that stuff which smells like cancer), it has pretty short shelf life after you unseal container, it will solidify eventually even if closed with that plastic cover. I believe thats why initially it was made as a single use item.
Ideally would be to make it as perishable food: as long as it sealed in its container it has infinite shelf life, once you unseal it - it become perishable with timer of 3-6 weeks until it become unusable.

Adding more types of glue is a good thing tho.

File: data/json/items/generic.json (hunk 1 above)
Comment (Holli-Git) [reply data/json/items/generic.json:324]: I'm unsure of an appropriate material. Would water be ok?

File: data/json/items/generic.json (hunk 2 above)
Comment (Holli-Git) [reply data/json/items/generic.json:329]: Shit, meant 5mm^2

**Comment by Holli-Git** [comment]
> If by "super glue" means some sort of cyanoacrylate glue(that stuff which smells like cancer), it has pretty short shelf life after you unseal container, it will solidify eventually even if closed with that plastic cover. I believe thats why initially it was made as a single use item. Ideally would be to make it as perishable food: as long as it sealed in its container it has infinite shelf life, once you unseal it - it become perishable with timer of 3-6 weeks until it become unusable.

Eh no, that'd be a clunky way to do it. I tried doing it so that when its out in the open it'd dry, but that didn't work

File: data/json/items/generic.json (hunk 1 above)
Comment (Karol1223) [reply data/json/items/generic.json]: > I'm unsure of an appropriate material. Would water be ok?

Probably not at all, honestly. Now that I think about it, plastic might be the closest thing we have in terms of materials...

**Comment by KittyTac** [comment]
> > If by "super glue" means some sort of cyanoacrylate glue(that stuff which smells like cancer), it has pretty short shelf life after you unseal container, it will solidify eventually even if closed with that plastic cover. I believe thats why initially it was made as a single use item. Ideally would be to make it as perishable food: as long as it sealed in its container it has infinite shelf life, once you unseal it - it become perishable with timer of 3-6 weeks until it become unusable.
> 
> Eh no, that'd be a clunky way to do it. I tried doing it so that when its out in the open it'd dry, but that didn't work

Why wouldn't it be like rotting food? Resealing glue doesn't do much to stop it from solidifying.

**Comment by Holli-Git** [comment]
> Why wouldn't it be like rotting food? Resealing glue doesn't do much to stop it from solidifying.

Because superglue doesn't rot, and that would confuse a lot of people. Freezing or cooling also wouldn't stop it

File: data/json/items/containers/containers.json [hunk 3]
```diff
@@ -678,6 +678,64 @@
...
+        "append": true
+      }
+    ]
+  },
+  {
+    "id": "small_squeeze_tube",
+    "type": "GENERIC",
+    "category": "container",
+    "name": { "str": "small squeeze tube" },
+    "description": "A small squeeze tube, holds 3 mL of liquid.  It cannot be refilled after its empty.",
```
Comment (TheShadowFerret) [inline data/json/items/containers/containers.json:705]:
```suggestion
    "description": "A small squeeze tube, holds 3 mL of liquid.  It cannot be refilled after it's empty.",
```

**Pull request closed without merging**